pkg/sensors/tracing/lsm_test.go

```diff
@@ -94,24 +100,93 @@ spec:
 	}
 	lsmChecker := ec.NewProcessLsmChecker("lsm-file-checker").
 		WithFunctionName(sm.Suffix("file_open")).
-		WithProcess(ec.NewProcessChecker().WithBinary(sm.Full("/usr/bin/cat"))).
+		WithProcess(ec.NewProcessChecker().WithBinary(sm.Full(testBin))).
 		WithArgs(ec.NewKprobeArgumentListMatcher().
 			WithOperator(lc.Ordered).
 			WithValues(
-				ec.NewKprobeArgumentChecker().WithFileArg(ec.NewKprobeFileChecker().WithPath(sm.Full("/etc/passwd")))))
+				ec.NewKprobeArgumentChecker().WithFileArg(ec.NewKprobeFileChecker().WithPath(sm.Full(tempFile)))))
 	obs, err := observertesthelper.GetDefaultObserverWithFile(t, ctx, testConfigFile, tus.Conf().TetragonLib, observertesthelper.WithMyPid())
 	if err != nil {
 		t.Fatalf("GetDefaultObserverWithFile error: %s", err)
 	}
 	observertesthelper.LoopEvents(ctx, t, &doneWG, &readyWG, obs)
 	readyWG.Wait()
 
-	catCmd := exec.Command("/usr/bin/cat", "/etc/passwd")
+	testCmd := exec.Command(testBin, tempFile)
+
+	if err := testCmd.Run(); err != nil {
+		t.Fatalf("failed to run %s: %s", testCmd, err)
+	}
+
+	err = jsonchecker.JsonTestCheck(t, ec.NewUnorderedEventChecker(lsmChecker))
+	assert.NoError(t, err)
+}
 
-	if err := catCmd.Run(); err != nil {
-		t.Fatalf("failed to run %s: %s", catCmd, err)
+func TestLSMBOverrideAction(t *testing.T) {
+	if !bpf.HasLSMPrograms() || !kernels.EnableLargeProgs() {
+		t.Skip()
```

Comment:
should we somehow also check bpf is configured on running kernel in CONFIG_LSM ?